target/linux/ramips/image/Makefile

```diff
@@ -196,6 +207,10 @@ define Device/uimage-lzma-loader
   KERNEL := kernel-bin | append-dtb | lzma | loader-kernel | uImage none
 endef
 
+ifeq ($(SUBTARGET),mt7621)
+include mt7621.mk
+endif
+
```

Comment:
This not needed anymore? I don't see it on master

Reply:
Indeed, drop that.

Reply:
Sorry, wrong button ... LOL.

Dang it, I was sure I removed this - but I think in all the git-fun I had, it crept back in. Thanks for catching it. Removed!